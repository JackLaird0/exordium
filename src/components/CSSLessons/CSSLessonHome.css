.title-container {
  background: none;
  border: none;
  width: 100%;
  display: flex;
  justify-content: center; 
  margin-top: 20px;
} 

.title {
  font-size: 40px;
  font-weight: 300;
}

.body-container {
  width: 85%;
}

.content,
.intstructions {
  box-sizing: border-box;
  padding: 10px 20px;
}

.overview,
.selectors,
.lessons-games {
  font-weight: 100;
}

.text {
  line-height: 30px;
  font-size: 18px;
}

.intstructions {
  margin-bottom: 30px;
}

.material {
  display: grid;
  grid-template-columns: 1fr 1fr;
  width: 50%;
  height: 200px;
  background: rgb(202, 202, 202);
  margin: 40px auto;
  border-radius: 5px;
}

.material-title {
  font-size: 20px;
  border-bottom: 1px solid black;
  padding: 2px;
  margin-top: 10px;
}

.lesson-element {
  margin: 1%;
  text-decoration: none;
}

.material-box,
.games {
  display: flex;
  justify-content: flex-start;
  flex-direction: column;
  align-items: center;
}

.image-container {
  display: flex;
  justify-content: center;
  align-items: center;
}

.css-image {
  height: 300px;
  margin: 20px;
}