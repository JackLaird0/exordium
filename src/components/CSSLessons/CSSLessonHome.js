import React, { Component } from 'react';
import selector from '../../assets/css-rule.png';
import './CSSLessonHome.css';
import { NavLink } from 'react-router-dom';

export class CSSLessonHome extends Component {

  render() {
    return (
      <main>
        <header className="title-container">
          <h1 className="title">Welcome to CSS!</h1>
        </header>
        <div className="body-container">
          <div className="content">
            <h1 className="overview">Overview</h1>
            <p className="text">
              It’s a “style sheet language” that lets you style the HTML elements on your page. CSS works with HTML, but isn’t HTML. CSS controls the positioning, sizing, colors, and specific fonts on your page. There is a class and id attribute available to use on every html element. In addition to the plain old tag names themselves, these attributes allow you to create “targets” for both your css and javascript. They are hooks so that you can manipulate the look and behavior of your HTML elements.
            </p>
            <h1 className="selectors">CSS Selectors</h1>
            <p className="text">
              In order for CSS to work with your HTML, we need to use selectors. CSS selectors allow us to "select" an HTML element by its:
            </p>
            <div className="selectors-container">
              <li>Element Type: "element"</li>
              <li>Class Name: ".class-name"</li>
              <li>Id: "#id"</li>
            </div>
            <p className="text">
              Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page. Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!
            </p>
            <div className="image-container">
              <img 
                className="css-image" 
                src={selector} 
                alt="CSS Selector Anatomy" 
              />
            </div>
            <p className="text">
              Selecting HTML elements provides us with the power to create beautiful web applications. Now that we know how to properly select elements, let's get started!
            </p>
          </div>
          <div className="instructions">
            <h1 className="lessons-games">Lessons & Games</h1>
            <p className="text">
              To get started, choose one of the lessons below. After you have completed the lesson, play the game to test your knowledge!
            </p>
            <div className="material">
              <div className="material-box">
                <h3 className="material-title">Lessons</h3>
                <div className="lesson-element">FlexBox</div>
                <div className="lesson-element">CSS Grid</div>
                <div className="lesson-element">Margins</div>
                <div className="lesson-element">Fonts</div>
              </div>
              <div className="games">
                <h3 className="material-title">Games</h3>
                <NavLink
                  to='/lessons_css/01'
                  className="lesson-element"
                >
                  <div>FlexBox</div>
                </NavLink>
                <div className="lesson-element">CSS Grid</div>
                <div className="lesson-element">Margins</div>
                <div className="lesson-element">Fonts</div>
              </div>
            </div>
          </div>
        </div>
      </main>
    );
  }
}

export default CSSLessonHome;