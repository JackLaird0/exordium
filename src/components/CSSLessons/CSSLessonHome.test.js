import React from 'react';
import { shallow } from 'enzyme';
import CSSLessonHome from './CSSLessonHome';

describe('CSSLessonHome', () => {
  
  it('should match the snapshot', () => {
    let wrapper = shallow(<CSSLessonHome />)
    expect(wrapper).toMatchSnapshot();
  })
})