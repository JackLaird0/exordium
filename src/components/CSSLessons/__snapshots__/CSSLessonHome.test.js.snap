// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CSSLessonHome should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CSSLessonHome />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header>
          <h1>
                    Welcome to CSS!
          </h1>
</header>,
        <div
          className="basics"
>
          <h1>
                    Before you start, please read this brief overview
          </h1>
          <p>
                    In order for CSS to work with your HTML, we need to use selectors
          </p>
          <p>
                    CSS selectors allow us to "select" an HTML element by its:
          </p>
          <li>
                    Element Type
          </li>
          <li>
                    Class Name
          </li>
          <li>
                    Id
          </li>
          <p>
                    Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page
          </p>
          <p>
                    Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!
          </p>
          <img
                    alt="CSS Selector Anatomy"
                    src="css-rule.png"
          />
          <p>
                    Now that we know how to properly select elements, let's get started!
          </p>
</div>,
        <div
          className="instructions"
>
          <p>
                    To get started, choose one of the lessons below
          </p>
          <p>
                    After you have completed the lesson, play the game to test your knowledge!
          </p>
</div>,
        <div
          className="lessons"
>
          <div>
                    Lesson 01: FlexBox
          </div>
          <div>
                    Lesson 02: CSS Grid
          </div>
          <div>
                    Lesson 03: Margins
          </div>
          <div>
                    Lesson 04: Fonts
          </div>
</div>,
        <div
          className="games"
>
          <NavLink
                    activeClassName="active"
                    aria-current="page"
                    to="/lessons_css/01"
          >
                    <div>
                              Game 01: FlexBox
                    </div>
          </NavLink>
          <div>
                    Game 02: CSS Grid
          </div>
          <div>
                    Game 03: Margins
          </div>
          <div>
                    Game 04: Fonts
          </div>
</div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1>
            Welcome to CSS!
</h1>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Welcome to CSS!",
          },
          "ref": null,
          "rendered": "Welcome to CSS!",
          "type": "h1",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1>
              Before you start, please read this brief overview
</h1>,
            <p>
              In order for CSS to work with your HTML, we need to use selectors
</p>,
            <p>
              CSS selectors allow us to "select" an HTML element by its:
</p>,
            <li>
              Element Type
</li>,
            <li>
              Class Name
</li>,
            <li>
              Id
</li>,
            <p>
              Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page
</p>,
            <p>
              Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!
</p>,
            <img
              alt="CSS Selector Anatomy"
              src="css-rule.png"
/>,
            <p>
              Now that we know how to properly select elements, let's get started!
</p>,
          ],
          "className": "basics",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Before you start, please read this brief overview",
            },
            "ref": null,
            "rendered": "Before you start, please read this brief overview",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "In order for CSS to work with your HTML, we need to use selectors",
            },
            "ref": null,
            "rendered": "In order for CSS to work with your HTML, we need to use selectors",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "CSS selectors allow us to \\"select\\" an HTML element by its:",
            },
            "ref": null,
            "rendered": "CSS selectors allow us to \\"select\\" an HTML element by its:",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Element Type",
            },
            "ref": null,
            "rendered": "Element Type",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Class Name",
            },
            "ref": null,
            "rendered": "Class Name",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Id",
            },
            "ref": null,
            "rendered": "Id",
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page",
            },
            "ref": null,
            "rendered": "Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!",
            },
            "ref": null,
            "rendered": "Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "CSS Selector Anatomy",
              "src": "css-rule.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Now that we know how to properly select elements, let's get started!",
            },
            "ref": null,
            "rendered": "Now that we know how to properly select elements, let's get started!",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <p>
              To get started, choose one of the lessons below
</p>,
            <p>
              After you have completed the lesson, play the game to test your knowledge!
</p>,
          ],
          "className": "instructions",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "To get started, choose one of the lessons below",
            },
            "ref": null,
            "rendered": "To get started, choose one of the lessons below",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "After you have completed the lesson, play the game to test your knowledge!",
            },
            "ref": null,
            "rendered": "After you have completed the lesson, play the game to test your knowledge!",
            "type": "p",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              Lesson 01: FlexBox
</div>,
            <div>
              Lesson 02: CSS Grid
</div>,
            <div>
              Lesson 03: Margins
</div>,
            <div>
              Lesson 04: Fonts
</div>,
          ],
          "className": "lessons",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lesson 01: FlexBox",
            },
            "ref": null,
            "rendered": "Lesson 01: FlexBox",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lesson 02: CSS Grid",
            },
            "ref": null,
            "rendered": "Lesson 02: CSS Grid",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lesson 03: Margins",
            },
            "ref": null,
            "rendered": "Lesson 03: Margins",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Lesson 04: Fonts",
            },
            "ref": null,
            "rendered": "Lesson 04: Fonts",
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/lessons_css/01"
>
              <div>
                            Game 01: FlexBox
              </div>
</NavLink>,
            <div>
              Game 02: CSS Grid
</div>,
            <div>
              Game 03: Margins
</div>,
            <div>
              Game 04: Fonts
</div>,
          ],
          "className": "games",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <div>
                Game 01: FlexBox
</div>,
              "to": "/lessons_css/01",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Game 01: FlexBox",
              },
              "ref": null,
              "rendered": "Game 01: FlexBox",
              "type": "div",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Game 02: CSS Grid",
            },
            "ref": null,
            "rendered": "Game 02: CSS Grid",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Game 03: Margins",
            },
            "ref": null,
            "rendered": "Game 03: Margins",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Game 04: Fonts",
            },
            "ref": null,
            "rendered": "Game 04: Fonts",
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "main",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header>
            <h1>
                        Welcome to CSS!
            </h1>
</header>,
          <div
            className="basics"
>
            <h1>
                        Before you start, please read this brief overview
            </h1>
            <p>
                        In order for CSS to work with your HTML, we need to use selectors
            </p>
            <p>
                        CSS selectors allow us to "select" an HTML element by its:
            </p>
            <li>
                        Element Type
            </li>
            <li>
                        Class Name
            </li>
            <li>
                        Id
            </li>
            <p>
                        Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page
            </p>
            <p>
                        Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!
            </p>
            <img
                        alt="CSS Selector Anatomy"
                        src="css-rule.png"
            />
            <p>
                        Now that we know how to properly select elements, let's get started!
            </p>
</div>,
          <div
            className="instructions"
>
            <p>
                        To get started, choose one of the lessons below
            </p>
            <p>
                        After you have completed the lesson, play the game to test your knowledge!
            </p>
</div>,
          <div
            className="lessons"
>
            <div>
                        Lesson 01: FlexBox
            </div>
            <div>
                        Lesson 02: CSS Grid
            </div>
            <div>
                        Lesson 03: Margins
            </div>
            <div>
                        Lesson 04: Fonts
            </div>
</div>,
          <div
            className="games"
>
            <NavLink
                        activeClassName="active"
                        aria-current="page"
                        to="/lessons_css/01"
            >
                        <div>
                                    Game 01: FlexBox
                        </div>
            </NavLink>
            <div>
                        Game 02: CSS Grid
            </div>
            <div>
                        Game 03: Margins
            </div>
            <div>
                        Game 04: Fonts
            </div>
</div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1>
              Welcome to CSS!
</h1>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Welcome to CSS!",
            },
            "ref": null,
            "rendered": "Welcome to CSS!",
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1>
                Before you start, please read this brief overview
</h1>,
              <p>
                In order for CSS to work with your HTML, we need to use selectors
</p>,
              <p>
                CSS selectors allow us to "select" an HTML element by its:
</p>,
              <li>
                Element Type
</li>,
              <li>
                Class Name
</li>,
              <li>
                Id
</li>,
              <p>
                Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page
</p>,
              <p>
                Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!
</p>,
              <img
                alt="CSS Selector Anatomy"
                src="css-rule.png"
/>,
              <p>
                Now that we know how to properly select elements, let's get started!
</p>,
            ],
            "className": "basics",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Before you start, please read this brief overview",
              },
              "ref": null,
              "rendered": "Before you start, please read this brief overview",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "In order for CSS to work with your HTML, we need to use selectors",
              },
              "ref": null,
              "rendered": "In order for CSS to work with your HTML, we need to use selectors",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "CSS selectors allow us to \\"select\\" an HTML element by its:",
              },
              "ref": null,
              "rendered": "CSS selectors allow us to \\"select\\" an HTML element by its:",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Element Type",
              },
              "ref": null,
              "rendered": "Element Type",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Class Name",
              },
              "ref": null,
              "rendered": "Class Name",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Id",
              },
              "ref": null,
              "rendered": "Id",
              "type": "li",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page",
              },
              "ref": null,
              "rendered": "Once selected, we can use CSS to manipulate our selected element, changing it's apperance and position on the page",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!",
              },
              "ref": null,
              "rendered": "Here is a sample CSS selector.  Notice its structure and syntax.  This will be important moving forward!",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "CSS Selector Anatomy",
                "src": "css-rule.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Now that we know how to properly select elements, let's get started!",
              },
              "ref": null,
              "rendered": "Now that we know how to properly select elements, let's get started!",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <p>
                To get started, choose one of the lessons below
</p>,
              <p>
                After you have completed the lesson, play the game to test your knowledge!
</p>,
            ],
            "className": "instructions",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "To get started, choose one of the lessons below",
              },
              "ref": null,
              "rendered": "To get started, choose one of the lessons below",
              "type": "p",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "After you have completed the lesson, play the game to test your knowledge!",
              },
              "ref": null,
              "rendered": "After you have completed the lesson, play the game to test your knowledge!",
              "type": "p",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                Lesson 01: FlexBox
</div>,
              <div>
                Lesson 02: CSS Grid
</div>,
              <div>
                Lesson 03: Margins
</div>,
              <div>
                Lesson 04: Fonts
</div>,
            ],
            "className": "lessons",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lesson 01: FlexBox",
              },
              "ref": null,
              "rendered": "Lesson 01: FlexBox",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lesson 02: CSS Grid",
              },
              "ref": null,
              "rendered": "Lesson 02: CSS Grid",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lesson 03: Margins",
              },
              "ref": null,
              "rendered": "Lesson 03: Margins",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Lesson 04: Fonts",
              },
              "ref": null,
              "rendered": "Lesson 04: Fonts",
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavLink
                activeClassName="active"
                aria-current="page"
                to="/lessons_css/01"
>
                <div>
                                Game 01: FlexBox
                </div>
</NavLink>,
              <div>
                Game 02: CSS Grid
</div>,
              <div>
                Game 03: Margins
</div>,
              <div>
                Game 04: Fonts
</div>,
            ],
            "className": "games",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <div>
                  Game 01: FlexBox
</div>,
                "to": "/lessons_css/01",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Game 01: FlexBox",
                },
                "ref": null,
                "rendered": "Game 01: FlexBox",
                "type": "div",
              },
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Game 02: CSS Grid",
              },
              "ref": null,
              "rendered": "Game 02: CSS Grid",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Game 03: Margins",
              },
              "ref": null,
              "rendered": "Game 03: Margins",
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Game 04: Fonts",
              },
              "ref": null,
              "rendered": "Game 04: Fonts",
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "main",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
